google/cloud/security/inventory/inventory_loader.py

```diff
@@ -194,15 +192,15 @@ def _send_email(organization_id, cycle_time, cycle_timestamp, status, pipelines,
 
     for pipeline in pipelines:
         try:
-            pipeline['count'] = dao.select_record_count(
-                pipeline['pipeline'].RESOURCE_NAME,
+            pipeline.count = dao.select_record_count(
```

Comment:
Is it possible to set this from pipeline.run() without having the email function have to call back out to SQL?